package.json

```diff
@@ -18,7 +18,9 @@
     "build-serial": "lerna run --concurrency 1 --stream build",
     "start": "lerna run start --stream --parallel",
     "pre-commit:husky": "lerna run --concurrency 1 --stream precommit",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "clean:modules": "rimraf node_modules && rimraf packages/*/node_modules",
+    "r": "node ./scripts/run.js"
```

Comment:
What are these changes for?

Reply:
Each time when I pull from master I want to clean my modules and run npm install to make sure the lock files are correct. `npm run clean:modules` will clear all node_modules